src/storage/redis_metadata.h

```diff
@@ -197,3 +199,33 @@ class StreamMetadata : public Metadata {
   void Encode(std::string *dst) override;
   rocksdb::Status Decode(const std::string &bytes) override;
 };
+
+class SBChainMetadata : public Metadata {
```

Comment:
We'd better add an version to metadata to support future evolution

Reply:
Not sure what's SB short for which words.

Reply:
Sure, SB is short for Scalable BloomFilter as RedisBloom do.